geometry/vectors_3d.c

```diff
@@ -191,6 +191,29 @@ mat_3x3 get_cross_matrix(const vec_3d *a)
     return A;
 }
 
+/**
+ * Obtain the angle between two given vectors.
+ * @f[\alpha=acos\left(\frac{\vec{a} \cdot \vec{b}}{\lVert\vec{a}\rVert \cdot \lVert\vec{b}\rVert}\right)@f]
+ * @param[in] a first input vector
+ * @param[in] b second input vector
+ * @returns angle between  @f$\vec{a}@f$ and @f$\vec{b}@f$ in radians
+ */
+
+double get_angle(const vec_3d *a, const vec_3d *b)
```

Comment:
The indentation in this function (inside) is wrong. Please fix it.

Reply:
You meant that I used tabs instead of four spaces?
If yes, then the problem was resolved

Reply:
No, the indentation is wrong, but you must use TABS here instead.

Reply:
It seems the indentation is fine now. 🙂

Reply:
Thanks a lot for your patience - this is my first pull request on github 🙂